<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>InlineFind</class>
 <widget class="QWidget" name="InlineFind" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>56</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>1</hsizetype>
    <vsizetype>1</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" >
   <property name="margin" >
    <number>8</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QToolButton" name="buttonHide" >
     <property name="text" >
      <string>...</string>
     </property>
     <property name="icon" >
      <iconset resource="qtedit4.qrc" >:/images/fileclose.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Find:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="editSearchText" />
   </item>
   <item>
    <widget class="QPushButton" name="buttonNext" >
     <property name="text" >
      <string>Find next</string>
     </property>
     <property name="icon" >
      <iconset resource="qtedit4.qrc" >:/images/next.png</iconset>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonPrevious" >
     <property name="text" >
      <string>Find previous</string>
     </property>
     <property name="icon" >
      <iconset resource="qtedit4.qrc" >:/images/previous.png</iconset>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbMatchCase" >
     <property name="text" >
      <string>Match &amp;case</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Maximum</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources>
  <include location="qtedit4.qrc" />
 </resources>
 <connections/>
</ui>
